src/transaction/log_manager.c

```diff
@@ -11263,6 +11266,8 @@ cdc_get_recdes (THREAD_ENTRY * thread_p, LOG_LSA * undo_lsa, RECDES * undo_recde
 	    }
 	}
 
+      memcpy (reserve_log_page_p, log_page_p, IO_MAX_PAGE_SIZE);
```

Comment:
This memcpy() is only needed when the log record exists in the separated log page. How about executing memcpy() only in that case?

Reply:
The forw_lsa in the LOG_RECORD_HEADER is a good source for checking this. (I think)

Reply:
OK, It seems reasonable. Whenever it tries to find `recdes` for each log record, it seems unnecessary to perform the memcpy() operation. Thank you. I will try to find a solution.